### PR TITLE
Update edit bounty

### LINKED ISSUE #403: Update Edit Bounty
@pitoi When editing a bounty, unlike wizard, all field will be listed in one page.
Header with title, "Cancel" and "Save" button will be sticky on top, so it will be always available.
In this version, we are only offering option to assign developer, but that would be expanded with invite option as soon as that option is available.

Video preview

https://user-images.githubusercontent.com/13258550/209681033-191a2456-4686-46aa-b2e4-a7b69159d46a.mp4

Full page 

edit-bounty

### PR DESCRIPTION
**Pull request by cosmicpotato137**
## Describe your changes
I added a header option to formfields, and reformatted the bounties>>edit page.

## Issue ticket number and link
Closes #403 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
I don't think this change is breaking but it should be tested.

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

## TODO
- [x] Finish formatting the edit form
- [ ] enable the github-description switch, possibly needs functionality?
- [ ] fix the left/right buttons for navigating between bounties on the bounties page

### CODE REVIEW

**Comment by cosmicpotato137** [comment]
@kevkevinpal regarding the prettier check issue: I ran both `yarn prettier` and `yarn prettier:check` successfully on my local machine, but I'm still having problems. Any thoughts?

**Comment by cosmicpotato137** [comment]
Finished what I can do for now! I also fixed the edit page accessed via the user's profile page by extension. The main container overflows and I still don't know how to fix it without breaking anything else.

**Comment by cosmicpotato137** [comment]
Changes made: 
- Updated edit bounty window on both home page and profile page
- Connected new bounty and edit bounty forms to the same schema and schemaData, formatted differently

Screenshot 2023-10-14 222306
Screenshot 2023-10-14 222244

File: frontend/app/src/components/form/index.tsx [hunk 1]
```diff
@@ -592,56 +607,45 @@ function Form(props: FormProps) {
...
+                ))}
+              </div>
             )}
 
             {/* make space at bottom for first sign up */}
             {buttonsOnBottom && !smallForm && <div style={{ height: 48, minHeight: 48 }} />}
+            {/* floating header buttons for edit bounty */}
             {!props?.newDesign && (
               <BWrap style={buttonAlignment} color={color}>
+                <h4 style={{ color: '#3C3D3F' }}>
```
Comment (kevkevinpal) [inline frontend/app/src/components/form/index.tsx:646]: we have a colors const file we should use it when defining colors

File: frontend/app/src/components/form/index.tsx [hunk 2]
```diff
@@ -157,7 +157,9 @@ function Form(props: FormProps) {
     });
   }
 
+  // check weather this is a multi-page form, filter by page
   let schema = props.paged ? props.schema?.filter((f: any) => f.page === page) : props.schema;
+  console.log('top of form');
```
Comment (kevkevinpal) [inline frontend/app/src/components/form/index.tsx:162]: remove log

**Review by kevkevinpal — APPROVED** [review]
looking good! going to merge in a bit

**Pull request merged**